src/ethereum/berlin/vm/__init__.py

```diff
@@ -14,16 +14,19 @@
 """
 
 from dataclasses import dataclass
-from typing import Dict, List, Optional, Set, Tuple, Union
+from typing import List, Optional, Set, Tuple, Union
 
 from ethereum.base_types import U256, Bytes, Bytes0, Bytes32, Uint, Uint64
 from ethereum.crypto.hash import Hash32
 
 from ..eth_types import Address, Log
-from ..state import State
+from ..state import State, account_exists_and_is_empty
+from ..utils.address import to_address
 
 __all__ = ("Environment", "Evm", "Message")
 
+RIPEMD160_ADDRESS = to_address(Uint(3))
```

Comment:
I think you can do this now:

```python
from .precompiled_contracts.mapping import RIPEMD160_ADDRESS
```

Reply:
Unfortunately this fails due to a circular import.

Reply:
Hm, that's a problem then. I'd rather not regress on #643. Could we move the `*_ADDRESS` constants to `__init__.py` maybe?